core/services/ardupilot_manager/main.py

```diff
@@ -15,6 +19,36 @@
 FRONTEND_FOLDER = Path.joinpath(Path(__file__).parent.absolute(), "frontend")
 
 
+parser = argparse.ArgumentParser(description="ArduPilot Manager service for Bluerobotics Companion")
+parser.add_argument("-v", "--verbose", help="increase output verbosity", action="store_true")
+
+args = parser.parse_args()
+
+if not args.verbose:
```

Comment:
Just to be sure, will this print errors and warnings by default (verbose as false) ?

Reply:
Yes. Verbose here is in the meaning that more information regarding each log message will be outputed (module and code line).

As an open source project with no sensitive information being exposed, I assume we should log debugs and warnings by default.

Reply:
just to clear, the only difference between verbose and not verbose is only the log format and not the output of such messages by category ?

Reply:
Yes.

Reply:
So, this is not what verbose means generally. 
I would recommend to make the 'pretty' format default and remove the verbose command at all since this does not change the verbosity and only the format.

Reply:
Ok, but the different format adds relevant information for debug.
Do you don't think we should completely remove it or have some suggestion on how to better implement that?

Reply:
Oh, ok. So to remove the current default (which does not have the module name and code line)?

Reply:
yes

Reply:
Done.